src/macros/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "macros"
```

Comment:
I am going to be annoying, can it be at the top level? being a sub-crate inside src is weird.